syft/frameworks/torch/hook/hook_args.py

```diff
@@ -40,7 +40,11 @@
     forward_func[crypten.mpc.MPCTensor] = (
         lambda i: i.child if hasattr(i, "child") else ().throw(PureFrameworkTensorFoundError)
     )
-    backward_func[crypten.mpc.MPCTensor] = lambda i, **kwargs: i.wrap(**kwargs)
+
```

Comment:
MPCTensor or never wrapped into another tensor, that's why you removed `backward_func[crypten.mpc.MPCTensor]` ?

Reply:
Put it back